ESP32_AP-Flasher/src/language.cpp

```diff
@@ -33,9 +33,33 @@ String languageNoDays[] = {"Søndag", "Mandag", "Tirsdag", "Onsdag", "Torsdag",
 String languageNoMonth[] = {"Januar", "Februar", "Mars", "April", "Mai", "Juni", "Juli", "August", "September", "Oktober", "November", "Desember"};
 /*END Norwegian language section END*/
 
-String* languageDaysShort[] = {languageEnDaysShort, languageNlDaysShort, languageDeDaysShort, languageNoDaysShort};
-String* languageDays[] = {languageEnDays, languageNlDays, languageDeDays, languageNoDays};
-String* languageMonth[] = {languageEnMonth, languageNlMonth, languageDeMonth, languageNoMonth};
+/*CZ Czech language section*/
+String languageCzDaysShort[] = {"NE", "PO", "ÚT", "ST", "ČT", "PÁ", "SO"};
+String languageCzDays[] = {"Neděle", "Pondělí", "Úterý", "Středa", "Čtvrtek", "Pátek", "Sobota"};
+String languageCzMonth[] = {"Leden", "Únor", "Březen", "Duben", "Květen", "Červen", "Červenec", "Srpen", "Září", "Říjen", "Listopad", "Prosinec"};
+/*END Czech language section END*/
+
+/*SK Slovak language section*/
+String languageSkDaysShort[] = {"NE", "PO", "UT", "ST", "ŠT", "PI", "SO"};
+String languageSkDays[] = {"Nedeľa", "Pondelok", "Utorok", "Streda", "Štvrtok", "Piatok", "Sobota"};
+String languageSkMonth[] = {"Január", "Február", "Marec", "Apríl", "Máj", "Jún", "Júl", "August", "September", "Oktober", "November", "December"};
+/*END Slovak language section END*/
+
+/*PL Polish language section*/
+String languagePlDaysShort[] = {"Nd", "Pon", "Wt", "Śr", "Czw", "Pt", "Sob"};
```

Comment:
This would need testing on devices to verify if the output is still short enough and readable, but the Polish language requires those abbreviations to end with a dot, like "pon.". Also they're supposed to be lower case.

Reply:
Ending with a "dot" is not necessarily required, it is permissible from the point of view of graphics to also use abbreviated days' names without this character. Alternatively, it is possible to completely simplify to just 2 characters: Po, Wt, Śr, Cz, Pt, Sa, Ni

Reply:
1) As part of simplifying the marking of "short name" days in Poland, an adjustment was made to make the style more uniform across languages. Change to: Po, Wt, Śr, Cz, Pt, Sa, Ni

2) Fixed a wrong "short name" day in Spanish language